core/src/test/java/org/apache/calcite/test/MaterializedViewSubstitutionVisitorTest.java

```diff
@@ -490,10 +490,12 @@
         "select count(*) + 1 as c, \"deptno\" from \"emps\" group by \"deptno\"")
         .withChecker(
             resultContains(""
-            + "LogicalCalc(expr#0..1=[{inputs}], expr#2=[1], "
-            + "expr#3=[+($t1, $t2)], C=[$t3], deptno=[$t0])\n"
-            + "  LogicalAggregate(group=[{1}], agg#0=[$SUM0($2)])\n"
-            + "    EnumerableTableScan(table=[[hr, MV0]])"))
+                + "LogicalCalc(expr#0..1=[{inputs}], expr#2=[1], expr#3=[+($t1, $t2)], C=[$t3], "
+                + "deptno=[$t0])\n"
+                + "  LogicalAggregate(group=[{0}], agg#0=[$SUM0($1)])\n"
```

Comment:
Do we have to add `LogicalCalc`? Is it possible to be compatible with the original? If `LogicalCalc is added, it may not be the optimal `RelNode`, which will increase the cost of calculation in physical execution.